docs/use_cases/personalized_search.md

```diff
@@ -6,31 +6,29 @@ Imagine a world where your online searches return results that truly understand
 
 ### How does it search?
 
 The power of vector embeddings lies in their ability to quantify the similarity between two vectors. This is done using a distance metric, with cosine similarity being one of the most commonly used metrics. Simply put, cosine similarity measures how close two vectors are to each other, helping to determine how similar two pieces of data are. It returns relevant results even when the exact terms aren't present in the query. This is made possible by the general-purpose nature of vector embeddings, which can represent almost any form of data, from text to images to audio.
 
 When vectorizing data, it's crucial to remember that embedding models have a limit on the maximum input length. The twelve best-performing models, based on the [Massive Text Embedding Benchmark (MTEB)](https://huggingface.co/spaces/mteb/leaderboard), are limited to an input size of 512 tokens, while the 13th best has an exceptional input size limit of 8192 tokens.
 
-For example, consider a user searching for a news article on a particular topic. If the article is longer than the model's maximum context length, the search system would need to split the article into smaller pieces or use a sliding window approach to capture the entire document in a single vector. This ensures that the search results are accurate and relevant, regardless of the length of the source document.
+Let's think about the previous example of searching for books again. If the book is longer than the model's maximum context length, the search system would need to split the book into smaller pieces or use a sliding window approach to capture the entire text in a single vector. This ensures that the search results are accurate and relevant, regardless of the length of the source text.
 
 ## But it's not only text that you can search!
 
 Contrary to popular belief, vector embeddings are not only applicable to unstructured data. In fact, they have always been integral to deep learning models and can be used to derive insights from structured data as well.
 
-For instance, consider a database of customer transactions. Each transaction can be represented as a vector, with each dimension representing a different attribute such as the transaction amount, date, or product category. By comparing these vectors, the search system can identify patterns or anomalies that would be difficult to spot with traditional search methods.
+In the book store example, we could represent each transaction as a vector, with each dimension representing a different attribute such as the transaction amount, date, or product category. By comparing these vectors, the search system can identify patterns or anomalies that would be difficult to spot with traditional search methods.
 
 ## Great! But what can I use to get started?
 
```

Comment:
Pls make the following changes - per my earlier review:

line 35:
instead of "a vector database, which is a system designed to store and perform semantic search at scale, you..."
say "a vector database – a system designed to store and perform semantic search at scale – you..."

instead of "compare the query vector with the vectors stored..."
say "compare the query vector with vectors stored..."

instead of "Furthermore vector databases"
say "Furthermore, vector databases"